recipes/friendly-remote-shell

```diff
@@ -0,0 +1,4 @@
+(friendly-remote-shell :repo "p3r7/friendly-remote-shell"
```

Comment:
This repository doesn't seem to exist at the moment. :)

Reply:
See my comment bellow.